src/components/dynamicOffline.js

```diff
@@ -0,0 +1,7 @@
+import dynamic from 'next/dynamic';
+
+const dynamicOffline = dynamic(import('./offline'), {
```

Comment:
Is there a particular reason to use dynamic imports ?

Reply:
The offline module is currently using the navigator.onLine event. But it's not working with the ssr, because there is no navigator. So I decided to use the dynamic module to skip the ssr and avoid this error.

It's just a wrapper around the react detect offline module.

Reply:
Got it ! So when server side renderer, does it render both ? None ? As online ?